repose-aggregator/extensions/api-validator/src/main/java/org/openrepose/filters/apivalidator/ValidatorConfigurator.java

```diff
@@ -135,6 +136,11 @@ private String getPath(String path, String configRoot) {
     }
 
     private String getWadlPath(String uri, String configRoot) {
-        return new File(configRoot, uri).toURI().toString();
+        //If the wadl path is already absolute, just return that rather than prepending the config root
+        if(Paths.get(uri).isAbsolute()) {
```

Comment:
what happened to using [this](http://docs.oracle.com/javase/7/docs/api/java/net/URI.html#resolve(java.lang.String))?